webapp/packages/plugin-administration/src/Administration/AdministrationToolbarStyles.m.css

```diff
@@ -9,3 +17,44 @@
 .administration .button {
   padding: 0 16px;
 }
+
+.tabList {
+  composes: theme-background-surface theme-text-on-surface theme-border-color-background from global;
+}
+.baseTab {
+  composes: theme-ripple theme-background-background theme-ripple-selectable from global;
+  color: inherit;
+}
+.slideElement {
+  composes: theme-background-secondary theme-text-on-secondary from global;
+}
+.captureView {
+  composes: theme-background-secondary theme-text-on-secondary from global;
+  display: flex;
+  flex-direction: row;
+  flex: 1;
+  overflow: hidden;
+}
+.tabList {
+  width: 240px;
+  padding-top: 16px;
+  border-right: 2px solid;
+}
+.contentContainer {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  overflow: auto;
+}
+.slideBox {
+  flex: 1;
+}
+.content {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  overflow: auto;
+}
```

Comment:
can be replaced with Container

Reply:
it brakes the layout. lets cover this next time when we dont have this much changes